client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -3,23 +3,79 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
 
 class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
+	state = {
+		city: '',
+		name: '',
+		stateLoc: '',
+		street: '',
+		zip: '',
+	};
+
+	setInput = field => event => {
+		this.setState( { [ field ]: event.target.value } );
+	};
+
 	render() {
 		const { translate } = this.props;
 		const headerText = translate( 'Add a business address.' );
 		const subHeaderText = translate(
 			'Enter your business address to have a map added to your website.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		const fields = [
+			[ 'city', 'City' ],
+			[ 'name', 'Business Name' ],
+			[ 'stateLoc', 'State' ],
+			[ 'street', 'Street Address' ],
+			[ 'zip', 'ZIP Code' ],
+		];
+		const formFields = fields.map( ( [ field, fieldText ] ) => {
```

Comment:
I'd probably just call the `fields.map` at the bottom, without declaring a variable, like this:

```
{ fields.map(...) }
```

If you feel it overcomplexes the `render` method, you might want to move it to another method.